src/Scripting/Core/Hosting/AssemblyLoader/DesktopAssemblyLoaderImpl.cs

```diff
@@ -15,14 +15,14 @@ public DesktopAssemblyLoaderImpl(InteractiveAssemblyLoader loader)
             : base(loader)
         {
             _assemblyResolveHandlerOpt = loader.ResolveAssembly;
-            CorLightup.Desktop.AddAssemblyResolveHandler(_assemblyResolveHandlerOpt);
+            CoreLightup.Desktop.AddAssemblyResolveHandler(_assemblyResolveHandlerOpt);
```

Comment:
I didn't understand why we can't eliminate the light up class for scripting too.

Reply:
The scripting code still depends on the assembly resolving process but is not multi-targeted (it's a netstandard 2.0 project). Scripting doesn't run on CoreClr at the moment hence this isn't actually a problem. But I was unsure how to unwind this code correctly. I filed the following bug to followup and it's linked to the commits that updated scripting

https://github.com/dotnet/roslyn/issues/31762